### PR TITLE
Add new Hippa Hitrust v9.2 compliance queries #56

### PR DESCRIPTION
**Pull request by krishna5891**
### Checklist
- [x] Issue(s) linked

### CODE REVIEW

File: query/cosmosdb/cosmosdb_use_virtual_service_endpoint.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
+select
+  -- Required Columns
+  cosmos.id as resource,
+  case
+    when document_endpoint like '%Microsoft.AzureCosmosDB%' then 'ok'
+    else 'ok'
+  end as status,
+  case
+    when document_endpoint like '%Microsoft.AzureCosmosDB%'  then cosmos.name || ' assoicated with "Microsoft.AzureCosmosDB" service endpoint.'
```
Comment (khushboo9024) [inline query/cosmosdb/cosmosdb_use_virtual_service_endpoint.sql:9]: We need to check the `virtual_network_rules` column.
```   
 "policyRule": {
      "if": {
        "allOf": [
          {
            "field": "type",
            "equals": "Microsoft.DocumentDB/databaseAccounts"
          },
          {
            "field": "Microsoft.DocumentDB/databaseAccounts/virtualNetworkRules[*].id",
            "exists": "false"
          }
        ]
      },
      ```

File: query/appservice/appservice_use_virtual_service_endpoint.sql [hunk 2]
```diff
@@ -0,0 +1,19 @@
+select
+  -- Required Columns
+  env.id as resource,
+  case
+    when internal_load_balancing_mode like '%Microsoft.Web%' then 'ok'
+    else 'alarm'
+  end as status,
+  case
+    when internal_load_balancing_mode like '%Microsoft.Web%'  then env.name || ' assoicated with "Microsoft.Web" service endpoint.'
```
Comment (khushboo9024) [inline query/appservice/appservice_use_virtual_service_endpoint.sql:9]: We need to check `Vnet Connection` for app service ,for this we need to add the api call `Get Vnet Connection` in  table `azure_app_web_app`. Please check this once.
```
"policyRule": {
      "if": {
        "allOf": [
          {
            "field": "type",
            "equals": "Microsoft.Web/sites"
          },
          {
            "field": "kind",
            "like": "app*"
          }
        ]
      },
      "then": {
        "effect": "[parameters('effect')]",
        "details": {
          "type": "Microsoft.Web/sites/virtualNetworkConnections",
          "existenceCondition": {
            "field": "Microsoft.Web/sites/virtualnetworkconnections/vnetResourceId",
            "exists": "true"
          }
        }
        ```

File: query/storage/storage_account_use_virtual_service_endpoint.sql [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    else 'alarm'
+  end as status,
+  case
+    when endpoints ->> 'service' like '%Microsoft.Storage%' then storage_account_subnet.name || ' assoicated with virtual service ' || subnet.service_endpoints || '.'
+    else storage_account_subnet.name || ' not assoicated with virtual service endpoint.' 
+  end as reason,
+  -- Additional Dimensions
+  resource_group,
+  sub.display_name as subscription
+from
```
Comment (khushboo9024) [inline query/storage/storage_account_use_virtual_service_endpoint.sql:23]: Getting repetitive result for this, please check once.

File: query/network/network_security_group_not_configured_gateway_subnets.sql [hunk 4]
```diff
@@ -0,0 +1,19 @@
+select
+  -- Required Columns
+  subnet.id resource,
+  case
+    when nat_gateway_id is null then 'skip'
```
Comment (khushboo9024) [inline query/network/network_security_group_not_configured_gateway_subnets.sql:5]: we have to check if the the subnet with name equals to `GatewaySubnet` is assigned with `networkSecurityGroup id ` then is should throw error, please check once.
```       {
            "field": "type",
            "equals": "Microsoft.Network/virtualNetworks/subnets"
          },
          {
            "field": "name",
            "equals": "GatewaySubnet"
          },
          {
            "field": "Microsoft.Network/virtualNetworks/subnets/networkSecurityGroup.id",
            "exists": "true"
          }
          ```

File: query/compute/compute_jit_access_protected.sql [hunk 5]
```diff
@@ -0,0 +1,35 @@
+with compute as (
```
Comment (khushboo9024) [inline query/compute/compute_jit_access_protected.sql:1]: file name should be `compute_vm_jit_access_protected.sql

**Review by rajlearner17 — APPROVED** [review]
LGTM

**Pull request merged**